test_runner/regress/test_bad_connection.py

```diff
@@ -1,10 +1,13 @@
 import random
 import time
 
+import psycopg2.errors
```

Comment:
Seems like this could just be import psycopg2?

Reply:
Why import more when you can import less?

Reply:
Also mypy complains if you remove `.errors`.